src/meta.rs

```diff
@@ -135,13 +130,21 @@ impl Meta {
                                                     Err(_) => MetaConfigObject::Unknown,
                                                 }
                                             }
```

Comment:
there is a panic on line 88 if the metabucket is empty
```
thread 'tokio-runtime-worker-2' panicked at 'called `Option::unwrap()` on a `None` value', src/libcore/option.rs:345:21
note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.
stack backtrace:
  9: core::option::Option<T>::unwrap
             at /rustc/3c235d5600393dfe6c36eeed34042efad8d4f26e/src/libcore/macros.rs:12
  10: minsql::meta::Meta::load_config_from_metabucket::{{closure}}
             at src/meta.rs:88
```

Reply:
this is fixed on #105